http.go

```diff
@@ -92,7 +103,7 @@ func (client *Client) Request(rawPayload []byte, method, route string, query url
 				return nil, nil, err
 			}
 		}
-		resp, err := (&http.Client{}).Do(callSummary.HTTPRequest)
+		resp, err := DefaultHTTPClient.Do(callSummary.HTTPRequest)
```

Comment:
No worries. I've changed the variable to be non-exported, so it can't be changed outside of the package, and there is no code that will currently modify the default in the package. The client-specific one will still be used should there be one.